pkg/expapi/register.go

```diff
@@ -28,6 +28,7 @@ func init() {
 // Adds the list of known types to api.Scheme.
 func addKnownTypes() {
 	api.Scheme.AddKnownTypes("",
+		&Deployment{},
```

Comment:
I think that we also need to define DeploymentList - this is necessary to support List operation.